navigator.sh

```diff
@@ -187,27 +196,86 @@ navigator() {
         return 0
     fi
     
-    # if [[ -z $directories ]]; then
-    #     echo No subdirectories to navigate to
-    #     return 0
-    # fi
-
-    directory_chooser $1 > >( { column -t -s $'\t'; echo "Enter the directory number (s to stop)- "; } )
+    directory_chooser $1 > >( { column -t -s $'\t'; printf "Enter the directory number (s to stop)\n${GREEN}|-> ${DEFAULT}"; } )
 
     read -r number
+	
+    if [[ "$number" == "s" ]]; then
+    	cd $PWD
+	exec $SHELL
```

Comment:
Doing this and exiting instead of a simple `return 0` yields unexpected results if user is doing this in a subshell. For example, if I use fish as my default shell and spawn a bash session to use `nn`, then whenever I stop the script, it spawns a new fish subshell instead of simply continuing the bash session. If I wish to use the `nn` command again, I then need to exit the fish subshell and spawn another bash session.

Reply:
The suggested changes have been made.